### ISSUE TITLE
Getting KeyError: 'location' for 'https://wuxiaworld.online/'

### ISSUE THREAD

**Issue by dipu-bd** [Anorov/cloudflare-scrape#236]
```
Python 3.7.1 (v3.7.1:260ec2c36a, Oct 20 2018, 14:57:15) [MSC v.1915 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import cfscrape
>>> cfscrape.create_scraper().get('https://wuxiaworld.online/')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\Dipu\AppData\Local\Programs\Python\Python37\lib\site-packages\requests\sessions.py", line 546, in get
    return self.request('GET', url, **kwargs)
  File "C:\Users\Dipu\AppData\Local\Programs\Python\Python37\lib\site-packages\cfscrape\__init__.py", line 83, in request
    resp = self.solve_cf_challenge(resp, **kwargs)
  File "C:\Users\Dipu\AppData\Local\Programs\Python\Python37\lib\site-packages\cfscrape\__init__.py", line 130, in solve_cf_challenge
    redirect_location = urlparse(redirect.headers["Location"])
  File "C:\Users\Dipu\AppData\Local\Programs\Python\Python37\lib\site-packages\requests\structures.py", line 52, in __getitem__
    return self._store[key.lower()][1]
KeyError: 'location'
```

```
C:\Users\Dipu>pip show cfscrape
Name: cfscrape
Version: 2.0.0
Summary: A simple Python module to bypass Cloudflare"s anti-bot page. See https://github.com/Anorov/cloudflare-scrape for more information.
Home-page: https://github.com/Anorov/cloudflare-scrape
Author: Anorov
Author-email: anorov.vorona@gmail.com
License: UNKNOWN
Location: c:\users\dipu\appdata\local\programs\python\python37\lib\site-packages
Requires: requests
Required-by:
```

**Comment by Anorov** [comment]
Please try upgrading with `pip install -U cfscrape`. This is very likely being caused by a captcha challenge - upgrading will give you a captcha-related error but won't resolve the problem. We're still working on how to hopefully reduce the rate at which captcha challenges are presented.

**Comment by Anorov** [comment]
Duplicate of #235.
